ui/pages/swaps/view-quote/view-quote.test.js

```diff
@@ -87,7 +88,7 @@ describe('ViewQuote', () => {
       getByTestId('main-quote-summary__exchange-rate-container'),
     ).toMatchSnapshot();
     expect(getByText('Estimated gas fee')).toBeInTheDocument();
-    expect(getByText('0.00544 ETH')).toBeInTheDocument();
+    expect(getByText('0.00008 ETH')).toBeInTheDocument();
```

Comment:
just curious why did this change?

Reply:
I've changed mock data with `feeEstimate` as part of this PR and because of that it's showing a little different estimate.